modules/helpers/stuff.js

```diff
@@ -0,0 +1,24 @@
+// From https://github.com/django/django/blob/b9cf764be62e77b4777b3a75ec256f6209a57671/django/contrib/admin/static/admin/js/urlify.js
+
```

Comment:
really? "stuff" ? We need more descriptive file names than that @Heyjp !

Reply:
Is there any way we can have slugs be created on update if they dont already exist? So that we can create slugs for older content

Reply:
`stuff.js` is very descriptive! It contains everything I don't have a name for. 

I did make a name change for this and we can definitely update on entity updates. Also made the change you requested in the meeting where the slug changes if the entity title changes.